scan/lib/scan/test_command_generator.rb

```diff
@@ -63,7 +63,9 @@ def pipe
         # During building we just show the output in the terminal
         # Check out the ReportCollector class for more xcpretty things
         formatter = []
-        if ENV.key?("TRAVIS")
+        if Scan.config[:formatter]
+          formatter << "-f `#{Scan.config[:formatter]}`"
```

Comment:
I don't think so: https://github.com/kattrali/xcpretty-travis-formatter#how-it-works